app/templates/event/partials/edit_event_details_tab.html

```diff
@@ -27,6 +27,25 @@
   <input type="text" id="invited" name="invited" placeholder="Invited emails, separated by commas">
 </div>
 
+<div class="shared_list" id="SharedList">
+  <div class="card card-body" id="SharedList">
+    <input type="text" class="form-control" name="shared_list_title" placeholder="Title">
+    <div id="Items">
+      <div class="form_row" id="Item">
+        <input type="text" class="form-control" name="item_name" placeholder="Item name">
+        <input type="text" class="form-control" name="item_amount" placeholder="Amount">
+        <input type="text" class="form-control" name="item_participant" placeholder="Participant">
+        <button type="button" class="btn btn-labeled btn-danger" id="btnRemoveItem" onclick="removeItem(this.id)">
```

Comment:
You should avoid adding JS to your HTML, in order to separate the logic from the view.

Please use JavaScript's `addEventHandler` instead.

Reply:
I'm now using addEventHandler :)